modules/serverless-logic/pages/job-services/core-concepts.adoc

```diff
@@ -31,7 +31,7 @@ For example, every time the workflow execution reaches a state with a configured
 
 image::job-services/Time-Based-States-And-Job-Service-Interaction.png[]
 
-To set up this integration you can use different xref:job-services/quarkus-extensions.adoc#job-service-quarkus-extensions[communication alternatives], that must be configured by combining the Job Service and the Quarkus Workflow Project configurations.
+To set up this integration you can use different xref:use-cases/advanced-developer-use-cases/job-service/quarkus-extensions.adoc#job-service-quarkus-extensions[communication alternatives], that must be configured by combining the Job Service and the Quarkus Workflow Project configurations.
```

Comment:
Since we do not prod jobs service, maybe we can cut this?

Reply:
Removed